gateleen-delegate/src/main/java/org/swisspush/gateleen/delegate/DelegateHandler.java

```diff
@@ -33,21 +33,21 @@
  * delegate.<br>
  * <code>PUT /gateleen/server/delegate/v1/delegates/user-zip-copy/definition</code>
  * <pre>
-    {
-        "methods": [ "PUT", "DELETE" ],
-        "pattern": "([^/]*)/(.*)",
-        "requests": [
-            {
-                 "headers" : [],
-                 "uri": "/gateleen/server/copy",
-                 "method": "POST",
-                 "payload": {
-                    "source": "/gateleen/$1?expand=100&amp;zip=true",
-                    "destination": "/gateleen/zips/users/$1.zip"
-                }
-            }
-        ]
-    }
+ {
+ "methods": [ "PUT", "DELETE" ],
+ "pattern": "([^/]*)/(.*)",
+ "requests": [
+ {
+ "headers" : [],
+ "uri": "/gateleen/server/copy",
+ "method": "POST",
+ "payload": {
+ "source": "/gateleen/$1?expand=100&amp;zip=true",
+ "destination": "/gateleen/zips/users/$1.zip"
+ }
+ }
+ ]
+ }
```

Comment:
??

Reply:
These were commits for another PR. They are removed again.